src/gui/activitywidget.cpp

```diff
@@ -588,8 +588,8 @@ void ActivitySettings::slotRefresh(AccountState *ptr)
 
 void ActivitySettings::slotRegularNotificationCheck()
 {
-    AccountManager *am = AccountManager::instance();
-    foreach (AccountStatePtr a, am->accounts()) {
+    const auto accounts = AccountManager::instance()->accounts();
```

Comment:
Its not obvious that in that case accounts  is a const reference so it looks like an explicit copy for no reason.
Any reason for that, besides an unrequired limit of 80 chars :)

Reply:
You're completely right, I pushed the fix.